screens/auth/SignUpScreen.js

```diff
@@ -228,37 +255,121 @@ export default class SignUp extends React.Component {
+            <Caption
+              color={
+                this.state.firstNameBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }>
+              Name
+            </Caption>
+            <Input
+              onBlur={() => this.onBlur(0)}
+              onFocus={() => this.onFocus(0)}
+              placeholder="First Name"
+              onChangeText={text => this.setState({ firstName: text })}
+              value={this.state.firstName}
+              borderColor={
+                this.state.firstNameBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }
+            />
+            <InputNoticeContainer>
+              <Caption color={Colors.secondaryText}>
+                Note: this is how clerks will greet you!
+              </Caption>
+            </InputNoticeContainer>
+          </InputContainer>
+          <InputContainer>
+            <Input
+              onBlur={() => this.onBlur(1)}
+              onFocus={() => this.onFocus(1)}
+              placeholder="Last Name"
+              onChangeText={text => this.setState({ lastName: text })}
+              value={this.state.lastName}
+              borderColor={
+                this.state.lastNameBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }
+            />
+          </InputContainer>
+
+          <InputContainer>
+            <Caption
+              color={
+                this.state.phoneNumBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }>
+              Phone Number
+            </Caption>
+            <Input
+              onBlur={() => this.onBlur(2)}
+              onFocus={() => this.onFocus(2)}
+              placeholder="Phone Number"
+              onChangeText={text => this.setState({ phoneNumber: text })}
+              value={this.state.phoneNumber}
+              keyboardType="number-pad"
+              maxLength={10}
+              borderColor={
+                this.state.phoneNumBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }
+            />
+          </InputContainer>
+
+          <InputContainer>
+            <Caption
+              color={
+                this.state.passwordBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }>
+              Password
+            </Caption>
+            <Input
+              onBlur={() => this.onBlur(3)}
+              onFocus={() => this.onFocus(3)}
+              placeholder="Password"
+              secureTextEntry
+              onChangeText={text => this.setState({ password: text })}
+              value={this.state.password}
+              borderColor={
+                this.state.passwordBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }
```

Comment:
Since you already made the InputNoticeContainer above for the note with Name, why don't you do the same here with the password requirements "Must be 8-20 characters long "